packages/generator-liferay-bundle/src/utils/ProjectAnalyzer.js

```diff
@@ -117,6 +131,29 @@ export default class ProjectAnalyzer {
 		return undefined;
 	}
 
+	/**
+	 * Get the path to the preferences file.
+	 * @return {string}
+	 */
+	get preferencesFilePath() {
+		const fs = this._generator.fs;
+
+		let preferences = prop.get(
```

Comment:
Seems this `let` should be `const`.
```suggestion
		const preferences = prop.get(
```

Reply:
:+1: